guides/common/attributes.adoc

```diff
@@ -1,5 +1,5 @@
-:ProjectVersion: 2.0
-:KatelloVersion: 3.15
+:ProjectVersion: 2.3
+:KatelloVersion: 3.18
```

Comment:
This is the nightly version so shouldn't this be 2.4 and Katello 4.0?

Reply:
Yes I will add a new commit once we agree, I want to merge this into 2.3 branch.

Reply:
FYI: nightly is about to bump to 2.5. Would you also branch 2.4?

Reply:
From what I understand, Katello releases come slightly after a Foreman release. Is it possible to have a branch, for example, for Foreman 2.4 & Katello 3.18 AND Foreman 2.4 and Katello 4?

Reply:
No, we only had 1 example where there was Katello 3.9 and Katello 3.10 on a single Foreman release but I don't really see that happening again. The biggest difference between 3.9 and 3.10 was EL8 support because it missed the 3.9 deadline. Overall it was a painful experience. I certainly hope we can avoid it.

Reply:
Ok, even better. Less complications. Thank you @ekohl